addons/web_editor/static/src/scss/wysiwyg.scss

```diff
@@ -296,8 +296,19 @@ body .modal {
             }
 
             .o_we_attachment_selected {
-                border-color: $o-brand-primary;
-                box-shadow: 0px 0px 2px 2px $o-brand-primary;
+                border: 3px solid $o-brand-primary;
+                &:before {
+                    content: "\f00c";
+                    @include o-position-absolute($top: 5px, $left: 5px);
+                    width: 1.2rem;
+                    height: 1.2rem;
+                    background-color: $o-brand-primary;
+                    font-family: 'FontAwesome';
+                    color: white; 
+                    border-radius: 50%;
+                    text-align: center;
+                    z-index: 1;
+                }
```

Comment:
Can't you share that code somehow? If it is the same design as in theme preview, I don't expect a copy/paste but sharing the code. Create a scss mixin in web_editor.variables.scss and use at both places